packages/rspack-plugin/package.json

```diff
@@ -20,13 +20,15 @@
   },
   "dependencies": {
     "@rsdoctor/core": "workspace:*",
-    "@rsdoctor/sdk": "workspace:*",
     "@rsdoctor/graph": "workspace:*",
+    "@rsdoctor/sdk": "workspace:*",
     "@rspack/core": "0.5.1",
+    "loader-utils": "^2.0.4",
```

Comment:
can we use the latest version (v3)

Reply:
Some api has changed, later I will change this to version 3 and chang the apis.

Reply:
> can we use the latest version (v3)

Some api has changed, later I will change this to version 3 and chang the apis.

Reply:
ok